### PR TITLE
Post-fix slot to match slot naming convention

### PR DESCRIPTION
**Pull request by donaldkibet**
### What does this PR do?

- [x] Post fix `slot` to extensionSlotName to comply with conventions

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -40,17 +40,17 @@ function setupOpenMRS() {
...
       {
         id: 'registration-nav-link',
-        slot: 'nav-menu',
+        slot: 'nav-menu-slot',
         load: getAsyncLifecycle(() => import('./nav-link'), options),
       },
       {
         id: 'patient-photo-widget',
-        slot: 'patient-photo',
+        slot: 'patient-photo-slot',
```
Comment (HerbertYiga) [inline src/index.ts:48]: @donaldkibet thanks

**Review by HerbertYiga — APPROVED** [review]

File: src/index.ts [hunk 2]
```diff
@@ -40,12 +40,12 @@ function setupOpenMRS() {
...
       {
         id: 'registration-home-link',
-        slot: 'home-page-buttons',
+        slot: 'homepage-dashboard-slot',
         load: getAsyncLifecycle(() => import('./home-link'), options),
       },
       {
         id: 'registration-nav-link',
-        slot: 'nav-menu',
+        slot: 'nav-menu-slot',
```
Comment (FlorianRappl) [inline src/index.ts]: You can remove this item.

We now have it in the top nav bar.

Comment (FlorianRappl) [reply src/index.ts]: Thanks! :beers:

**Review by FlorianRappl — APPROVED** [review]

File: src/index.ts [hunk 3]
```diff
@@ -40,14 +40,9 @@ function setupOpenMRS() {
...
         load: getAsyncLifecycle(() => import('./home-link'), options),
       },
-      {
-        id: 'registration-nav-link',
-        slot: 'nav-menu',
-        load: getAsyncLifecycle(() => import('./nav-link'), options),
-      },
       {
         id: 'patient-photo-widget',
         slot: 'patient-photo',
```
Comment (FlorianRappl) [inline src/index.ts]: Should that be `patient-photo-slot`?

**Review by FlorianRappl — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 3 above)
Comment (donaldkibet) [reply src/index.ts]: Yes it should be that, i have updated the name

**Review by FlorianRappl — APPROVED** [review]

**Pull request merged**